src/main/kotlin/Person.kt

```diff
@@ -0,0 +1,6 @@
+data class Person(
+    val name: String,
+    val company: String?,
+    val skills: Skills?,
+    val languages: Map<String, Int>
```

Comment:
Why not use this chance to understand the differences between sealed class and enum class and use them in your next mission? 🙂

https://kotlinlang.org/docs/sealed-classes.html

```kotlin
sealed interface Skill {
    data class Soft : Skill
    data class Hard : Skill
}
```